src/wallet/wallet.cpp

```diff
@@ -3131,7 +3131,7 @@ bool CWallet::CreateTransaction(interfaces::Chain::Lock& locked_chain, const std
 
     // Before we return success, we assume any change key will be used to prevent
     // accidental re-use.
-    reservekey.KeepKey();
+    reservedest.KeepDestination();
```

Comment:
This change should be in the previous commit.

Reply:
done